pennylane/utils.py

```diff
@@ -235,30 +236,61 @@ def __init__(self, queue, observables, parameters=None):
                     self._grid[w] = []
 
                 # Add the operation to the grid, to the end of the specified wire
-                self._grid[w].append([idx, cmd])
+                self._grid[w].append(cmd)
 
         self._graph = nx.DiGraph()
 
-        # iterate over each wire in the grid
-        for _, cmds in self._grid.items():
+        # Iterate over each wire in the grid
+        for cmds in self._grid.values():
             if cmds:
                 # Add the first operation on the wire to the graph
                 # This operation does not depend on any others
-                attrs = cmds[0][1]._asdict()
-                self._graph.add_node(cmds[0][0], **attrs)
+                self._graph.add_node(cmds[0].idx, **cmds[0]._asdict())
```

Comment:
Just curious, what happened to the 1 that appeared in `attrs = cmds[0][1]._asdict()`?